string request
---
int32 obsPeriodInMs
string[] propertiesId
string[] objectsId
string[] agentsId

